dj_cqrs/management/commands/cqrs_dead_letters.py

```diff
@@ -0,0 +1,127 @@
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
+
+import ujson
+from django.core.management.base import BaseCommand, CommandError
+
+from dj_cqrs.dataclasses import TransportPayload
+from dj_cqrs.registries import ReplicaRegistry
+from dj_cqrs.transport.rabbit_mq import RabbitMQTransport
+from dj_cqrs.transport import current_transport
+from dj_cqrs.utils import get_expires_datetime
+
+
+class RabbitMQTransportService(RabbitMQTransport):
+
+    @classmethod
+    def get_consumer_settings(cls):
+        return cls._get_consumer_settings()
+
+    @classmethod
+    def get_common_settings(cls):
+        return cls._get_common_settings()
+
+    @classmethod
+    def create_connection(cls, host, port, creds, exchange):
+        return cls._create_connection(host, port, creds, exchange)
+
+    @classmethod
+    def declare_queue(cls, channel, queue_name):
+        return channel.queue_declare(queue_name, durable=True, exclusive=False)
+
+    @classmethod
+    def nack(cls, channel, delivery_tag, payload=None):
+        return cls._nack(channel, delivery_tag, payload)
+
+
+class Command(BaseCommand):
+    help = 'CQRS dead letters queue management commands'
+
+    def add_arguments(self, parser):
+        command = parser.add_subparsers(dest='command')
+        command.required = True
+        command.add_parser('retry', help='Retry all dead letters.')
+        command.add_parser('bulk_dump', help='Dumps all dead letter to stdout.')
```

Comment:
I really don't like to discuss naming, but I have to ask here if simple `dump` is better. 
I would also ask to talk with Stanislav on help descriptions, because for example 'Removes all dead letters' is misleading a bit.

P.S. There is no dot in the end in 1 of 3 helps.

Reply:
We already have cqrs_bulk_dump command. Wanted to make naming more consistent.
Will rename to dump.